src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private transient int size;
+    private transient Node<K, V>[] table;
```

Comment:
The `table` field is declared as `transient`, which means it won't be serialized. Ensure this is intentional, as it might affect the behavior if you plan to serialize instances of `MyHashMap`.